tcms/static/js/utils.js

```diff
@@ -100,6 +100,19 @@ function update_component_select_from_product() {
     }
 }
 
+function updateTestCaseSelectFromProduct() {
+    const updateCallback = (data) => {
+        updateSelect(data, '#id_select_test_plan', 'plan_id', 'name')
+    };
+
+    const productId = $('#id_product').val();
+    if (productId) {
+        jsonRPC('Component.filter', {product: product_id}, updateCallback);
```

Comment:
This should filter `TestPlan` shouldn't it? Also naming !!!!

Reply:
Copy-paste leftover

Reply:
Lack of testing !